build_scripts/build_usd.py

```diff
@@ -539,8 +539,15 @@ def DownloadURL(url, context, force, extractDir = None,
     Returns the absolute path to the directory where files have 
     been extracted."""
     with CurrentWorkingDirectory(context.srcDir):
-        # Extract filename from URL and see if file already exists. 
-        filename = url.split("/")[-1]       
+        # Extract filename from URL.
+        filename = url.split("/")[-1]
```

Comment:
I'd consider if it would be better to use `urllib.parse.urlparse` to breakdown the `url` into its components (https://docs.python.org/3/library/urllib.parse.html).  That should be more robust than splitting on `/`.

It might also make things more robust to just use the full `path` component of the URL when creating a target to download the files rather than trying to decompose based on the naming conventions of particular repositories.  I'm not sure if there's any drawbacks to that.

Reply:
The only drawback that I can see with the name as the path is if the filename become too long, it might not save properly.